mobile/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     compile 'com.android.support:multidex:1.0.2'
     compile 'com.google.code.gson:gson:2.8.0'
     fullCompile 'com.google.android.gms:play-services-gcm:11.4.2'
+    fullCompile 'com.google.android.gms:play-services-maps:11.4.2'
```

Comment:
11.8.0 is available. Do you want to bump both gms packages here or make a new PR for that?

Reply:
I'd prefer a new PR for it, I think. I'm not sure whether there are any side effects.